create-event/src/components/Root.jsx

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+
+import React, { Component } from 'react'
+
+import Header from '../components/Header'
+import Routes from '../components/Routes'
+import DialogTextInput from '../components/DialogTextInput'
+import SnackbarContainer from '../containers/SnackbarContainer'
+
+class Root extends Component {
+  componentWillMount() {
+    this.props.fetchUser('sk_live_67c2aa1275dbb5e0822ba9bfa97ea132')
+  }
+
+  render() {
+    const { user, paths, components, fetchUser } = this.props
+
+    if (!user.id) {
+      return (
+        <DialogTextInput
```

Comment:
at the moment this dialog flashes on page load, if you are not going to let a user input an api key I would delete this